pkg/kube/crds.go

```diff
@@ -264,6 +268,46 @@ func RegisterAppCRD(apiClient apiextensionsclientset.Interface) error {
 	return RegisterCRD(apiClient, name, names, columns, &validation, jenkinsio.GroupName)
 }
 
+// RegisterApplicationCRD ensures that the CRD is registered for Applications
+func RegisterApplicationCRD(apiClient apiextensionsclientset.Interface) error {
+	name := "applications." + jenkinsio.GroupName
+	names := &v1beta1.CustomResourceDefinitionNames{
+		Kind:       "Application",
+		ListKind:   "ApplicationList",
+		Plural:     "applications",
+		Singular:   "application",
+		ShortNames: []string{"application"},
+	}
+	columns := []v1beta1.CustomResourceColumnDefinition{
+		{
+			Name:        "Name",
```

Comment:
There is automatically a name column (the name of the resource) so having a second name column is confusing.